base/loading.jl

```diff
@@ -511,73 +585,72 @@ module_uuid(m::Module) = ccall(:jl_module_uuid, UInt64, (Any,), m)
 
 isvalid_cache_header(f::IOStream) = 0 != ccall(:jl_read_verify_header, Cint, (Ptr{Void},), f.ios)
 
-function cache_dependencies(f::IO)
```

Comment:
I also changed what it does, since Base doesn't need this functionality. I can put back `cache_dependencies` with some tests and just not export it since looking at AWSLambda.jl, it looks like it was using it correctly.